osquery/tables/system/windows/services.cpp

```diff
@@ -110,13 +110,14 @@ static inline Status getService(const SC_HANDLE& scmHandle,
   r["name"] = SQL_TEXT(svc.lpServiceName);
   r["display_name"] = SQL_TEXT(svc.lpDisplayName);
   r["status"] = SQL_TEXT(kSvcStatus[svc.ServiceStatusProcess.dwCurrentState]);
-  r["pid"] = INTEGER(svc.ServiceStatusProcess.dwProcessId);
+  r["pid"] = SQL_TEXT(svc.ServiceStatusProcess.dwProcessId);
```

Comment:
Why are you changing this to a `SQL_TEXT` here? Is this necessary? A `pid` imo should be an `unsigned long` on windows if we can help it.

Reply:
The reason for using a text field rather than an integer is to keep a more uniform interface for all OS and init systems, because it is more natural to leave the field unpopulated (instead of using a magic value).

If this is an issue, I can revert it. Thanks for the feedback!